### ISSUE TITLE
Right connector symbol: Uart2 RX/TX swapped?

### ISSUE THREAD

**Issue by michael-p** [bitcraze/crazyflie2-exp-template-electronics#2]
Hi!
The right side connector symbol has Uart2 TX on Pin 1 and RX on Pin 2, however, looking at the documentation image in this repo Pin 1 should be RX and 2 TX. Could you please clarify which one is correct?

Thanks a lot!
Michael

**Comment by knmcguire** [comment]
Hi! The documentation image one should be the one that is correct. I've tried to make a deck myself the other day with those exact rx and tx so I'm pretty sure that is the right one :)

**Comment by michael-p** [comment]
Ok thanks! Would probably be a really good idea to fix the symbol in this repo, otherwise people relying on the template will have a pretty bad time ;)

**Comment by knmcguire** [comment]
True, but I think that it is still correct though. The documentation shows the **RX** and **TX** of the crazyflie, however, the TX (transmitting data) of the crazyflie has to go to the RX (receiving data) on the expansion deck and visa versa.

**Comment by michael-p** [comment]
That was my first thought as well, however, if that is correct than Uart1 on the left side connector is wrong :) (there, pins are not swapped)

**Comment by knmcguire** [comment]
ahhhh oke, yeah that should be fixed! I'll assign somebody to it
